### PR TITLE
Configuration for the Probot Stale GitHub app

### LINKED ISSUE #1790: ⚠ Automatically close old Issues and PRs
There is a suggestion to use this bot: https://github.com/marketplace/stale :robot: to automatically mark MFEM issues and PRs as stale, and close them after a period of inactivity.

Please use this issue to discuss the pros and cons of the proposal (particularly if you oppose it) and the specific bot settings.

Here is what I (@tzanio) suggest:

```
# Number of days of inactivity before an issue becomes stale
daysUntilStale: 30
# Number of days of inactivity before a stale issue is closed
daysUntilClose: 7
# Issues with these labels will never be considered stale
exemptLabels:
  - bug
  - WIP
  - ready-for-review
  - in-review
  - in-next
# Label to use when marking an issue as stale
staleLabel: stale
# Comment to post when marking an issue as stale. Set to `false` to disable
markComment: >
  This issue has been automatically marked as stale because it has not had
  recent activity. It will be closed if no further activity occurs. Thank you
  for your contributions.
# Comment to post when closing a stale issue. Set to `false` to disable
closeComment: false
```

### PR DESCRIPTION
**Pull request by tzanio**
Resolves #1790.

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#1807](https://github.com/mfem/mfem/pull/1807) | @tzanio | @tzanio | @v-dobrev + @kmittal2 + @camierjs + @adrienbernede | 10/07/20 | 10/08/20 | 10/08/20 | |

### CODE REVIEW

**Review by adrienbernede — APPROVED** [review]
I approve!

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - WIP
+  - ready-for-review
+  - in-review
+  - in-next
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
```
Comment (kmittal2) [inline .github/stale.yml:17]: Is it worth mentioning here that the issue "will be closed in 7 days if no further activity occurs"?

**Review by kmittal2 — APPROVED** [review]
Looks good to me.

File: .github/stale.yml (hunk 1 above)
Comment (tzanio) [reply .github/stale.yml]: Addressed among other things in 0d9ad5bc2e5e659f43e1c92f38934e0350a3efa8

**Comment by camierjs** [comment]
LGTM

**Review by camierjs — APPROVED** [review]

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Pull request merged**